packages/connect/src/types/api/getFeatures.ts

```diff
@@ -1,4 +1,6 @@
 import type { CommonParams, Response } from '../params';
 import type { Features } from '../device';
 
-export declare function getFeatures(params?: CommonParams): Response<Features>;
+export type FeaturesParams = CommonParams;
```

Comment:
I talked about this with @marekrjpolak in the office and would argue that its better to have nominal types even in cased they are same (structurally). 

The reason for this is, that in case you want to extend them, you do it only in one place and you dont need to find few usages of `CommonParams` that are used for the context of the `FeaturesParams`.

Reply:
still I don't want to introduce this change here. please lets move it to a separate PR that we can iterate on. Otherwise we will never finish it and will stay somewhere half baked